main.py

```diff
@@ -0,0 +1,18 @@
+from pycodb.base import Base
+
+
+if __name__ == '__main__':
+
+    class Employee(Base):
+        __table_id__ = 'md96lqumeig8ue1'
+        __view_id__ = 'vwths26rv60mxd9e'
```

Comment:
Please don't keep your personal keys in the codebase. That's why we have .env git-ignored in the repo